src/index.d.ts

```diff
@@ -98,19 +104,29 @@ type ConnectData = {
 }
 
 type TransactionsOptions = {
-  institutionId: string;
+  institutions?: Array<string>;
   from?: string;
   to?: string;
   investments?: string;
   last?: string;
   limit?: number;
+  institutionId?: string;
 }
 
 type TransactionOptions = {
-  institutionId: string;
   txId: string;
 }
 
+type PositionsOptions = {
+  institutions?: Array<string>;
+  groupIds?: Array<string>;
```

Comment:
Same here, should it just be `groups`?

Reply:
I think better to use the same values as in API without overwriting then institutionIds etc.

Reply:
Yep, fixed